loadtest/store/memstore/random.go

```diff
@@ -162,16 +170,21 @@ func (s *MemStore) RandomUsers(n int) ([]model.User, error) {
 	s.lock.RLock()
 	defer s.lock.RUnlock()
 
-	if n > len(s.users) {
+	numUsers := len(s.users)
```

Comment:
Let's add the comment you mentioned here: https://github.com/mattermost/mattermost-load-test-ng/pull/265#issuecomment-623419534. This is a very important point.